spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -224,7 +224,7 @@ def paginated_collection(*args)
         let(:pagination) { paginated_collection(collection, pagination_total: true) }
 
         it "should show the total item counts" do
-          info = pagination.find_by_class('pagination_information').first.content.gsub('&nbsp;',' ')
+          info = pagination.find_by_class('pagination-information').first.content.gsub('&nbsp;',' ')
```

Comment:
Line is too long. [100/80]<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.